Services/User/Profile/classes/class.ilPersonalProfileGUI.php

```diff
@@ -527,7 +527,61 @@ protected function showUserAgreement()
         $this->tpl->setPermanentLink('usr', null, 'agreement');
         $this->tpl->printToStdout();
     }
-    
+
+    protected function showConsentWithdrawalConfirmation() : void
```

Comment:
IMO this does not reflect the conceptual state described here: https://docu.ilias.de/goto_docu_wiki_wpage_5120_1357.html

I cannot find any special handling for `LDAP` users.